pages/_app.tsx

```diff
@@ -1,8 +1,11 @@
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
+import Navbar from "../component/navbar"
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return <Navbar>
+    <Component {...pageProps} />
+    </Navbar>
```

Comment:
Can we edit it so that on the index route "/", the navbar doesn't get rendered? Take a look at [this](https://stackoverflow.com/questions/58013066/how-to-hide-navbar-header-in-login-page-in-nextjs)